README.rst

```diff
@@ -275,9 +275,10 @@ announcements you can join the |pyhf-announcements mailing list|_.
 Citation
 --------
 
-As noted in `Use and
-Citations <https://scikit-hep.org/pyhf/citations.html>`__, the preferred
-BibTeX entry for citation of ``pyhf`` is
+As noted in `Use and Citations <https://scikit-hep.org/pyhf/citations.html>`__,
+the preferred BibTeX entries for citation of ``pyhf`` includes both the `JOSS
+<https://joss.theoj.org/>`__ paper and the `Zenodo <https://zenodo.org/>`__
+archive:
```

Comment:
```suggestion
As noted in `Use and Citations <https://scikit-hep.org/pyhf/citations.html>`__,
the preferred BibTeX entries for citation of ``pyhf`` includes both the
`Zenodo <https://zenodo.org/>`__ archive and the
`JOSS <https://joss.theoj.org/>`__ paper:
```

As the render on the citations page shows the Zenodo entry first can we have the language match that?

![contradict_myself](https://user-images.githubusercontent.com/5142394/106951221-1a591b00-66f5-11eb-9034-5ffa16c1ff4b.png)